packages/tsconfig-reference/scripts/tsconfigRules.ts

```diff
@@ -248,34 +248,8 @@ export const defaultsForOptions = {
 };
 
 export const allowedValues = {
-  jsx: ["`react`", "`react-jsx`", "`react-jsxdev`", "`react-native`", "`preserve`"],
```

Comment:
These allowed values are currently in sync with `ts.buildOpts["jsx"].type.keys()`, but removing them ensures we'll pick up future updates?

Reply:
Yah, this seems reasonable to me